examples/xscope/host/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+cmake_minimum_required(VERSION 3.21)
+
+# Check if XMOS_TOOL_PATH is defined
+if(NOT DEFINED ENV{XMOS_TOOL_PATH})
+    message(FATAL_ERROR "XMOS_TOOL_PATH must be defined.")
+endif()
+
+# Compile for x86_64 on Mac as we can't support the M1 ARM architecture yet
+set(CMAKE_OSX_ARCHITECTURES
+    "x86_64"
+    CACHE INTERNAL "")
+
+set(CMAKE_BUILD_TYPE "Release")
+project(xscope_host_app)
+
+# Define path to lib_device_control
+set(DEVICE_CONTROL_PATH ${CMAKE_CURRENT_LIST_DIR}/../../../lib_device_control)
+
+# Define source files
+set(SOURCES
+    "${DEVICE_CONTROL_PATH}/host/device_access_xscope.c"
+    "${DEVICE_CONTROL_PATH}/host/util.c"
+    "src/host.c"
+)
+add_executable(xscope_host_app ${SOURCES})
+
+# Directories
+target_include_directories(
+  xscope_host_app
+    PRIVATE
+    src
+    ${DEVICE_CONTROL_PATH}/host
+    ${DEVICE_CONTROL_PATH}/api
+    ${DEVICE_CONTROL_PATH}/src
+    $ENV{XMOS_TOOL_PATH}/include
```

Comment:
is this needed?

Reply:
This is the only way I found to include the xscope dynamic file, let me know if there are better ways.